web/src/locales/da.json

```
@@ -111,38 +392,391 @@
   "unknown": "Ukendt",
+  "electricityStoredUsing": "er opbevaret vha.",
+  "emissionsStoredUsing": "er opbevaret vha.",
+  "electricityExportedTo": "er eksporteret til",
+  "emissionsExportedTo": "er eksporteret til",
+  "electricityImportedFrom": "er importeret fra",
+  "emissionsImportedFrom": "er importeret fra",
+  "electricityAvailableIn": "af elektricitet tilgængelig i",
+  "emissionsAvailableIn": "af udledningen af electricitet tilgængelig i",
+  "comesFrom": "kommer fra",
   "ofCO2eq": "af CO₂-ækvivalenter",
+  "theMap": {
+    "groupName": "Kortet",
+    "mapColors-question": "Hvad betyder farverne?",
+    "mapColors-answer": "Vi farver områderne på kortet svarende til udledningen af drivhusgas der bliver udledt pr enhed af electricitetsforbrug (CO₂-intensiteten). Jo grønnere, jo mere klimavenligt er det at bruge electriciteten i dette område.",
+    "mapArrows-question": "Hvad betyder pilene mellem områderne på kortet?",
+    "mapArrows-answer": "Pilene mellem de forskellige områder indikere fysisk strømning (import og eksport) af electricitet mellem områder: jo hurtigere de blinker, jo hurtigere er strømningen. Strømningen af electriciteten er vigitigt, da udledningen associeret med electricitets produktionen også importeres når strømmen importeres fra nabolandende.",
+    "mapSolarWindButtons-question": "Hvad gør “sol” og “vind” kanpperne på kortet?",
+    "mapSolarWindButtons-answer": "Disse kanpper viser realtids vind hastighed og solstyrke, hvilket giver en indikation på potentialet af at kunne omstille til sol og vind baserede energi alternativer i forskellige områder. Bemærk dog at dette kun er en overfladisk indikation, da det faktiske potentiale af at kunne installere vindmøller og solceller også baseres på mange andre faktorer.",
+    "mapNuclearColors-question": "Hvorfor er lande primært baseret på atomkraft som energikilde vist som grønne på kortet?",
+    "mapNuclearColors-answer": "Atomkrafts produktion har en meget lav CO₂-intensitet (<a href=\"https://en.wikipedia.org/wiki/Life-cycle_greenhouse-gas_emissions_of_energy_sources#2014_IPCC.2C_Global_warming_potential_of_selected_electricity_sources\" target=\"_blank\" >kilde</a>), og lande primært baseret på atomkraft som energikilde er derfor vist som grønne på kortet. Dette betyder ikke at der ikke er andre klima udfordringer associeret med atomkraft (ligesom dette også er tilfældet med andre energi produktions former), men disse udfordringer er ikke relaterede til udledningen af drivhusgasser og er derfor ikke udenfor hvad dette kort har at tilbyde.",
+    "mapColorBlind-question": "Jeg er farveblind. Hvordan kan jeg læse jeres kort?",
+    "mapColorBlind-answer": "Du kan ændre farverne på kortet ved at aktivere “farveblindhedstilstand“. Du kan finde den i højre side af kortet på computer og i indstillinger på mobil."
+  },
+  "mapAreas": {
+    "groupName": "Områder på kortet",
+    "noData-question": "Hvorfor er der ikke data tilgængeligt for mit område?",
```

changing this as it's a bit weird in Danish to ask about "my area" 😅 
```suggestion
    "noData-question": "Hvorfor er der ikke data tilgængeligt for alle områder?",
```